spec/services/moment_keywords_spec.rb

```diff
@@ -0,0 +1,34 @@
+describe MomentKeywords do
+  subject(:moment) {FactoryBot.build(:moment,
+                                     categories: [build(:category, name: 'free', description: 'Description')],
+                                     moods: [build(:mood, name: 'Name', description: 'Blog^^')],
+                                     strategies: [build(:strategy, name: 'Name', description: 'books@!##.')],
+                                     name: "ADDICTION",
+                                     why: "self-care.",
+                                     fix: "@Teachers!!" )}
```

Comment:
The most accepted style for multiline blocks is using `do...end`:

```suggestion
  subject(:moment) do
    FactoryBot.build(
      :moment,
      categories: [build(:category, name: 'free', description: 'Description')],
      moods: [build(:mood, name: 'Name', description: 'Blog^^')],
      strategies: [build(:strategy, name: 'Name', description: 'books@!##.')],
      name: "ADDICTION",
      why: "self-care.",
      fix: "@Teachers!!"
    )
  end
```